mobile-app/app/screens/AppNavigator/screens/Loans/components/VaultCardV2.tsx

```diff
@@ -0,0 +1,230 @@
+import BigNumber from "bignumber.js";
+import { ThemedTextV2, ThemedViewV2 } from "@components/themed";
+import { tailwind } from "@tailwind";
+import { translate } from "@translations";
+import { NavigationProp, useNavigation } from "@react-navigation/native";
+import { LoanParamList } from "@screens/AppNavigator/screens/Loans/LoansNavigator";
+import { loanTokensSelector, LoanVault } from "@store/loans";
+import {
+  LoanToken,
+  LoanVaultActive,
+} from "@defichain/whale-api-client/dist/api/loan";
+import { TouchableOpacity, View } from "react-native";
+import { useVaultStatus } from "@screens/AppNavigator/screens/Loans/components/VaultStatusTag";
+import { useNextCollateralizationRatio } from "@screens/AppNavigator/screens/Loans/hooks/NextCollateralizationRatio";
+import { useLoanOperations } from "@screens/AppNavigator/screens/Loans/hooks/LoanOperations";
+import { VaultStatus } from "@screens/AppNavigator/screens/Loans/VaultStatusTypes";
+import { getPrecisedTokenValue } from "@screens/AppNavigator/screens/Auctions/helpers/precision-token-value";
+import { NumericFormat as NumberFormat } from "react-number-format";
+import { BottomSheetNavScreen } from "@components/BottomSheetWithNavV2";
+import { useSelector } from "react-redux";
+import { RootState } from "@store";
+import { TokenIconGroupV2 } from "@components/TokenIconGroupV2";
+import { LoanActionButton } from "./LoanActionButton";
+import { VaultSectionTextRowV2 } from "./VaultSectionTextRowV2";
+import { VaultBanner } from "./VaultBanner";
+
+export interface VaultCardProps extends React.ComponentProps<any> {
+  vault: LoanVault;
+  testID: string;
+  dismissModal: () => void;
+  expandModal: () => void;
+  setBottomSheetScreen: React.Dispatch<
+    React.SetStateAction<BottomSheetNavScreen[]>
+  >;
+  setSnapPoints: React.Dispatch<
+    React.SetStateAction<{
+      ios: string[];
+      android: string[];
+    }>
+  >;
+  onBottomSheetLoansTokensListSelect: ({
+    onPress,
+    loanTokens,
+  }: {
+    onPress: (item: LoanToken) => void;
+    loanTokens: LoanToken[];
+  }) => void;
+}
+
+export function VaultCardV2(props: VaultCardProps): JSX.Element {
```

Comment:
<img width="512" alt="image" src="https://user-images.githubusercontent.com/13292662/194453473-c688a5f9-2934-4652-b081-cb675c74c9a3.png">
1. Borrow Button is wrong color, will it be part of the circular lib PR?
2. Spacing is also off

Reply:
3. When using $, it should be 2 decimal places

Reply:
4. Liquidated Vault collateral is not displaying
5. Liquidated Vault color is always black
<img width="512" alt="image" src="https://user-images.githubusercontent.com/13292662/194455753-2791d01e-caa3-46e4-948a-c31d72bae7e9.png">

Reply:
Point 1,2,3,4 and 5 are done on Joshua's PR :D